assets/xml/objects/object_famos.xml

```diff
@@ -1,28 +1,30 @@
-        <Limb Name="object_famos_Standardlimb_003D00" Type="Standard" EnumName="OBJECT_FAMOS_LIMB_03" Offset="0x3D00" />
-        <Limb Name="object_famos_Standardlimb_003D0C" Type="Standard" EnumName="OBJECT_FAMOS_LIMB_04" Offset="0x3D0C" />
-        <Limb Name="object_famos_Standardlimb_003D18" Type="Standard" EnumName="OBJECT_FAMOS_LIMB_05" Offset="0x3D18" />
-        <Skeleton Name="object_famos_Skel_003D38" Type="Normal" LimbType="Standard" LimbNone="OBJECT_FAMOS_LIMB_NONE" LimbMax="OBJECT_FAMOS_LIMB_MAX" EnumName="object_famos_Limbs" Offset="0x3D38" />
-        <Animation Name="object_famos_Anim_003DC8" Offset="0x3DC8" />
-        <TextureAnimation Name="object_famos_Matanimheader_003E30" Offset="0x3E30" />
-        <TextureAnimation Name="object_famos_Matanimheader_003E38" Offset="0x3E38" />
+        <Animation Name="gFamosShakeAnim" Offset="0xF8" />
+        <DList Name="gFamosHelmetDL" Offset="0x12B0" />
+        <DList Name="gFamosMainBodyDL" Offset="0x1528" />
+        <DList Name="gFamosShieldDL" Offset="0x1C88" />
+        <DList Name="gFamosSwordDL" Offset="0x1DE8" />
+        <DList Name="gFamosLightSwitchDL" Offset="0x1FA8" />
+        <Texture Name="gFamosTLUT2068" OutName="tlut_002068" Format="rgba16" Width="4" Height="4" Offset="0x2068" />
+        <Texture Name="gFamosTLUT2088" OutName="tlut_002088" Format="rgba16" Width="4" Height="4" Offset="0x2088" />
+        <Texture Name="gFamosTLUT20A8" OutName="tlut_0020A8" Format="rgba16" Width="4" Height="4" Offset="0x20A8" />
+        <Texture Name="gFamosTLUT20C8" OutName="tlut_0020C8" Format="rgba16" Width="4" Height="4" Offset="0x20C8" />
+        <Texture Name="gFamosMainBodyTex" OutName="famos_main_body" Format="ci4" Width="64" Height="64" Offset="0x20E8" />
+        <Texture Name="gFamosBottomTex" OutName="famos_bottom" Format="ci4" Width="32" Height="32" Offset="0x28E8" />
+        <Texture Name="gFamosShieldTex" OutName="famos_shield" Format="ci4" Width="64" Height="64" Offset="0x2AE8" />
+        <Texture Name="gFamosSwordHiltTex" OutName="famos_sword_hilt" Format="ci4" Width="32" Height="32" Offset="0x32E8" />
+        <Texture Name="gFamosLightSwitchTex" OutName="famos_light_switch" Format="rgba16" Width="32" Height="32" Offset="0x34E8" />
+
+        <Limb Name="gFamosBodyLimb" Type="Standard" EnumName="FAMOS_LIMB_BODY" Offset="0x3CE8" />
+        <Limb Name="gFamosEmblemLimb" Type="Standard" EnumName="FAMOS_LIMB_EMBLEM" Offset="0x3CF4" />
+        <Limb Name="gFamosSwordLimb" Type="Standard" EnumName="FAMOS_LIMB_SWORD" Offset="0x3D00" />
+        <Limb Name="gFamosShieldLimb" Type="Standard" EnumName="FAMOS_LIMB_SHIELD" Offset="0x3D0C" />
+        <Limb Name="gFamosHeadLimb" Type="Standard" EnumName="FAMOS_LIMB_HEAD" Offset="0x3D18" />
+        <Skeleton Name="gFamosSkeleton" Type="Normal" LimbType="Standard" LimbNone="FAMOS_LIMB_NONE" LimbMax="FAMOS_LIMB_MAX" EnumName="ObjectFamosLimbs" Offset="0x3D38" />
```

Comment:
Since you use `FAMOS_LIMB_[whatever]` for the enum values, I'd drop the "Object" from the `EnumName` like this:
```suggestion
        <Skeleton Name="gFamosSkeleton" Type="Normal" LimbType="Standard" LimbNone="FAMOS_LIMB_NONE" LimbMax="FAMOS_LIMB_MAX" EnumName="FamosLimbs" Offset="0x3D38" />
```

Since we know this is a Death Armos, I think it'd be better to use `DEATH_ARMOS_LIMB_[whatever]` and then call it `DeathArmosLimbs`, but the consistency is the important point. If you *do* change the name like that, you probably want to swap all the `gFamos[whatever]` stuff in here to `gDeathArmos[whatever]`

Reply:
Until we decide when/if we change actor names from ripped names to official names, this just seems to make all variable names longer IMO.